libs/sdk-backend-tiger/src/convertors/toBackend/afm/tests/FilterConverter.test.ts

```diff
@@ -130,6 +130,10 @@ describe("tiger filter converter from model to AFM", () => {
                 "range measure value filter with crossed boundaries",
                 newMeasureValueFilter(ReferenceLdm.Won, "BETWEEN", 128, 64),
             ],
+            [
+                "ranking filter",
+                newRankingFilter(ReferenceLdm.Won, [ReferenceLdm.IsActive.attribute.displayForm], "TOP", 3),
+            ],
```

Comment:
i suggest to add extra test to cover ranking filter without attributes

Reply:
fixed